plugins/inputs/cisco_telemetry_mdt_netconf/README.md

```diff
@@ -0,0 +1,171 @@
+    update_trigger = "periodic"
+    period = "5s"
+    tags = ["/interfaces-ios-xe-oper:interfaces/interface/name"]
+
+  ## IOS-XE Subscription - on-change
+  [[inputs.cisco_telemetry_mdt_netconf.subscription_service.subscription]]
+    xpath_filter = "/cdp-ios-xe-oper:cdp-neighbor-details/cdp-neighbor-detail"
+    update_trigger = "on-change"
+    period = "0s"
+
+  ## Get operations
+  ## IOS-XE Get Request with filter
+  [[inputs.cisco_telemetry_mdt_netconf.get_service.get]]
+    xpath_filter = "/interfaces-state/interface[name='GigabitEthernet1']/oper-status"
+    period = "10s"
+    tags = ["/interfaces-state/interface/name"]
+
+  ## IOS-XE Get Request with filter and multiple tags
+  [[inputs.cisco_telemetry_mdt_netconf.get_service.get]]
+    xpath_filter = "/interfaces-state/interface[name='GigabitEthernet2']"
+    period = "10s"
+    tags = ["/interfaces-state/interface/name", "/interfaces-state/interface/if-index"]
+
+  ## IOS-XE Get Request without filter
+  [[inputs.cisco_telemetry_mdt_netconf.get_service.get]]
+    xpath_filter = "/memory-statistics/memory-statistic"
+    period = "10s"
+    tags = ["/memory-statistics/memory-statistic/name"]
+
+  ## Event notification subscription
+  ## NSO Event notfication subscription with a tag
+  [[inputs.cisco_telemetry_mdt_netconf.subscription_service.notification]]
+    stream = "ncs-alarms"
+    tags = ["alarm-notification/alarm-class"]
+```
+
+## Example Output
+
+```shell
+Cisco-IOS-XE-process-cpu-oper:cpu-usage/cpu-utilization,host=linux,path=Cisco-IOS-XE-process-cpu-oper:cpu-usage/cpu-utilization,source=3650,subscription=1 five_seconds=3i 1654898172903000000
+Cisco-IOS-XE-memory-oper:memory-statistics/memory-statistic,Cisco-IOS-XE-memory-oper:memory-statistics/memory-statistic/name=reserve\ Processor,host=linux,source=3650 total-memory=102404i,used-memory=92i,free-memory=102312i,lowest-usage=102312i,highest-usage=102312i 1654898176722493661
+Cisco-IOS-XE-interfaces-oper:interfaces/interface/statistics,host=linux,name=GigabitEthernet0/0,path=Cisco-IOS-XE-interfaces-oper:interfaces/interface/statistics,source=3650,subscription=2 discontinuity_time="2022-06-07T22:06:03.000763+00:00",in_octets=55043194i,in_unicast_pkts=580774i,in_broadcast_pkts=0i,in_multicast_pkts=0i,in_discards=0i,in_errors=0i,in_unknown_protos=0i,out_octets=9692848i,out_unicast_pkts=107693i,out_broadcast_pkts=0i,out_multicast_pkts=0i,out_discards=0i,out_errors=0i,rx_pps=5i,rx_kbps=18i,tx_pps=1i,tx_kbps=2i,num_flaps=0i,in_crc_errors=0i 1654898187900000000
+```
+
+## Metrics
+
+- Cisco-IOS-XE-process-cpu-oper:cpu-usage/cpu-utilization
+  - tags:
+    - host
+    - path
+    - source
+    - subscription
+  - fields:
+    - five_seconds (int, percent)
+
+- Cisco-IOS-XE-memory-oper:memory-statistics/memory-statistic
+  - tags:
+    - host
+    - source
+    - Cisco-IOS-XE-memory-oper:memory-statistics/memory-statistic/name (enforced tag)
```

Comment:
Hello @Hipska. Thank you for your review. The memory-statistics does not have a path and subscription because it is an output sample from the GET operation, not from a subscription.

Reply:
Oh okay, that makes sense. I was confused because it was listed between two other metrics that did come from subscription.

Reply:
Erm, no wait. This metric does come from a subscription:
```toml
    ## IOS-XE Subscription - periodic
    [[inputs.cisco_telemetry_mdt_netconf.subscription_service.subscription]]
      xpath_filter = "/memory-ios-xe-oper:memory-statistics/memory-ios-xe-oper:memory-statistic"
      update_trigger = "periodic"
      period = "10s"
      tags = ["/memory-ios-xe-oper:memory-statistics/memory-ios-xe-oper:memory-statistic/memory-ios-xe-oper:name"]
```

Reply:
The configuration has 2 memory-statistic samples, one with a subscription and one with get request. I've added the output samples independently of the config, the original config having been tested a long time ago.

What would be the recommendation here? To match all the sample config in the **Example output** section?

Reply:
Yes, see my next comment about this. It would be the most logic thing for a user seeing the sample configs and then in the documentation a sample output that would match the config..